docs/docs/configuration.md

```diff
@@ -112,6 +112,10 @@ Defaults to `true`.
 Create the virtualenv inside the project's root directory.
 Defaults to `false`.
 
+If set to `true`, the virtualenv wil be created and expected in a folder named `.venv` within the root directory of the project.
+
+If set to `None`, `poetry` will use the virtualenv from the `.venv` directory only if available.
```

Comment:
```suggestion
If not set explicitly, `poetry` will use the virtualenv from the `.venv` directory only if available.
```

Reply:
This might be confusing, because the default is `false`. One could interpret "not set explicitly" as "default not changed".

Reply:
The ens user should never set `None`. By default it is `false` because when we attempt to get the config we get `None` because it is not set. This distinction is something we should handle internally hence the suggestion.